src/elm/State.elm

```diff
@@ -14,7 +14,7 @@ import Types exposing (..)
 
 initModel : Model
 initModel =
-    { route = LandingRoute
+    { route = ThankYouRoute
```

Comment:
😄

Reply:
I swear I didn't do it on this one!! gah

Reply:
done